v1/contributors/serializers/contributor.py

```diff
@@ -5,8 +5,15 @@
 
 
 class ContributorSerializer(serializers.ModelSerializer):
+
     class Meta:
-        fields = 'pk', 'display_name', 'github_username', 'slack_username', \
-                 'created_date', 'modified_date'
+        fields = (
```

Comment:
just a suggestion, to not use alphabetic ordering here (or in such cases)
rather follow the "importance" order
created and modified are not that important
pk is the first and most important field ...
others - indeed, alphabetic

Reply:
How would you define "importance" so that it is not subjective? The main reason for alphabetic ordering is because it's easy for new developers to understand that standard and follow the pattern.